apps/chatter/consumers.py

```diff
@@ -2,11 +2,14 @@
 import json
 from channels.generic.websocket import AsyncWebsocketConsumer
 
+# WEBSOCKETS - The consumer is like the view for websockets.
+# It is registered in the app routing.py
 class ChatConsumer(AsyncWebsocketConsumer):
     async def connect(self):
-        # print('CONNECTION RECEIVED')
-        self.room_name = self.scope['url_route']['kwargs']['room_name']
-        self.room_group_name = 'chat_%s' % self.room_name
+        print('CONNECTION RECEIVED')
+        self.room_id = self.scope['url_route']['kwargs']['room_id']
+        self.room_group_name = 'chat_%s' % self.room_id
```

Comment:
Small note, but `"".format()` is the preferred way to format strings over the `%` operator.

https://www.python.org/dev/peps/pep-3101/